sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/schemas/SchemaResolver.java

```diff
@@ -193,7 +193,7 @@ private static InputStream getSchemaByURL(String urlStr) {
     try {
       return url.openStream();
     } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)